docs/source/secured_asset_transfer/secured_private_asset_transfer_tutorial.md

```diff
@@ -13,19 +13,19 @@ The private asset transfer scenario is bound by the following requirements:
 
 - An asset may be issued by the first owner's organization (in the real world issuance may be restricted to some authority that certifies an asset's properties).
 - Ownership is managed at the organization level (the Fabric permissioning scheme would equally support ownership at an individual identity level within an organization).
-- The asset identifier is a hash of the asset's immutable properties, and along with the current owner is stored as public channel data for all channel members to see.
```

Comment:
Thanks for the update.

Question - why are you removing the information about the asset hash throughout this document? It was intentionally documented as a concept for users to understand.

Here is the corresponding code that creates the hash:
https://github.com/hyperledger/fabric-samples/blob/main/asset-transfer-secured-agreement/chaincode-go/asset_transfer.go#L59-L62

Reply:
Thanks for pointing this out. I've restored the descriptions as my changes were more towards the command's inputs and their outputs.

Reply:
Looks like it is happening dur to the way type is declared in  https://github.com/hyperledger/fabric-samples/blob/main/asset-transfer-secured-agreement/chaincode-go/asset_transfer.go